app/src/main/java/com/wonjoong/android/sopthub/ui/home/ImageDataBindingAdapter.kt

```diff
@@ -0,0 +1,13 @@
+package com.wonjoong.android.sopthub.ui.home
+
+import android.widget.ImageView
+import androidx.databinding.BindingAdapter
+
+object ImageDataBindingAdapter {
+
+    @BindingAdapter("bind")
+    @JvmStatic
+    fun setImageViewResource(imageView: ImageView, resource: Int) {
+        imageView.setImageResource(resource)
+    }
```

Comment:
선생님 확장함수로
```suggestion
    fun ImageView.setImageViewResource(resource: Int) {
        setImageResource(resource)
    }
```
이렇게도 표현가능합니다~!!